audit-drivers.md

```diff
@@ -58,3 +58,36 @@ class MyCustomDriver implements AuditDriver
 ```
 
 > {tip} The `Database` driver is a good starting point to get ideas for a new custom driver implementation.
+
+## Defining the custom driver
+
+Here's how to set the custom driver `MyCustomDriver`:
+```php
+<?php
+namespace App\Models;
+
+use App\AuditDrivers\MyCustomDriver;
+use Jenssegers\Mongodb\Eloquent\Model;
+use OwenIt\Auditing\Audit as AuditTrait;
+use OwenIt\Auditing\Contracts\Audit as AuditContract;
+
+class Audit extends Model implements AuditContract
```

Comment:
The update you're suggesting doesn't seem to be correct, since the `auditDriver` attribute should be set in the `Auditable`, not in the `Audit`.